exp/api/v1alpha3/conversion_test.go

```diff
@@ -32,17 +32,17 @@ func TestFuzzyConversion(t *testing.T) {
 	g.Expect(AddToScheme(scheme)).To(Succeed())
 	g.Expect(v1alpha4.AddToScheme(scheme)).To(Succeed())
 
-	t.Run("for AzureMachinePool", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
-		Scheme: scheme,
-		Hub:    &v1alpha4.AzureMachinePool{},
-		Spoke:  &AzureMachinePool{},
-	}))
-
-	t.Run("for AzureManagedCluster", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
-		Scheme: scheme,
-		Hub:    &v1alpha4.AzureManagedCluster{},
-		Spoke:  &AzureManagedCluster{},
-	}))
+	//t.Run("for AzureMachinePool", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
```

Comment:
🙂 ?

Reply:
hah, thanks for catching that! I must be debugging something :)